CriticalMass/NetworkOperator.swift

```diff
@@ -57,7 +46,7 @@ struct NetworkOperator: NetworkLayer {
     private func dataTask(with request: URLRequest,
                           completion: @escaping ResultCallback<Data>) {
         networkIndicatorHelper.didStartRequest()
```

Comment:
The new tests aren't covering networkIndicator activation. Not sure if tests for that are needed though